bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -100,6 +118,38 @@ spec:
       # /env -> configmap env-overrides - debug overrides
       containers:
       {{ if eq .OVN_NODE_MODE "full" }}
+      {{if .ENABLE_OVN_NODE_PROXY}}
+      # ovnkube-node-proxy redirects ovn sbdb traffic to http proxy
+      - name: ovnkube-node-proxy
+        image: "{{.OvnImage}}"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -xe
+          if [[ -f "/env/${K8S_NODE}" ]]; then
+            set -o allexport
+            source "/env/${K8S_NODE}"
+            set +o allexport
+          fi
+          echo "$(date -Iseconds) - starting ovnkube-node-proxy"
+          # Don't use /etc/hosts to resolve OVN_SB_DB_ROUTE_HOST which points to 127.0.0.1 (see spec.hostAliases)
+          echo "hosts: dns files" >> /etc/nsswitch.conf
+          exec socat TCP-LISTEN:{{.OVN_SB_DB_ROUTE_LOCAL_PORT}},reuseaddr,fork PROXY:{{.HTTP_PROXY_IP}}:{{.OVN_SB_DB_ROUTE_HOST}}:{{.OVN_SB_DB_ROUTE_PORT}},proxyport={{.HTTP_PROXY_PORT}}
+        env:
+        - name: K8S_NODE
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        volumeMounts:
+        - mountPath: /env
+          name: env-overrides
+        terminationMessagePolicy: FallbackToLogsOnError
+        resources:
+          requests:
+            cpu: 10m
+            memory: 300Mi
```

Comment:
This seems like a lot -- do we need 300 MiB? Can we kick this down a bit?

Reply:
Fixed